tools/barista/src/markdown-custom-elements-ignore.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Prevents the markdown parser from wrapping custom "ba-" tags into a paragraph. */
+export const baElementBlockIgnore = (state, startLine, endLine, silent) => {
```

Comment:
NIT: why we are using a const arrow function? Consistency would be normal function

Reply:
can not answer - implemented by @ffriedl89 & @tomheller